python/lsst/ts/MTAOS/model.py

```diff
@@ -352,14 +433,182 @@ async def run_wep(self, visit_id, extra_id, config, **kwargs):
             self.log.debug(
                 f"Processing MainCamera corner wavefront sensor on image {visit_id}."
             )
+            raise NotImplementedError(
+                "LSSTCam Corner wavefront sensor processing not implemented yet."
+            )
         else:
-            # Will have to verify images are ComCam and raise an exception if
-            # they are main camera. Main camera intra/extra data will be
-            # processed exclusively using OCPS.
-            self.log.debug(f"Processing intra/extra pair: {visit_id}/{extra_id}.")
+            # If data is intra/extra it must be ComCam at this point. Main
+            # camera intra/extra data will be processed exclusively using OCPS.
+            self.log.debug(
+                f"Processing intra/extra pair: {visit_id}/{extra_id}. Expecting ComCam data."
+            )
+
+            await self.process_comcam(
+                intra_id=visit_id, extra_id=extra_id, config=config
+            )
+
+    async def process_comcam(self, intra_id, extra_id, config):
+        """Process ComCam intra/extra focual images.
+
+        Parameters
+        ----------
+        intra_id : `int`
+            Id of the intra-focal image.
+        extra_id : `int`
+            Id of the extra-focal image.
+        config : `dict`
+
+        """
+        self.log.debug(f"Processing ComCam intra/extra pair: {intra_id}/{extra_id}.")
+
+        wep_configuration = self.generate_wep_configuration(
+            instrument="comcam", reference_id=intra_id, config=config
+        )
+
+        comcam_config_file = tempfile.NamedTemporaryFile(suffix=".yaml")
+
+        comcam_config_file.write(yaml.safe_dump(wep_configuration).encode())
+
+        comcam_config_file.flush()
+
+        self.log.debug(f"Pipeline configuration in {comcam_config_file.name}.")
+
+        run_pipetask_cmd = writePipetaskCmd(
+            self.data_path,
+            self.run_name,
+            self.pipeline_instrument["comcam"],
+            "refcats," + self.collections,
+            pipelineYaml=comcam_config_file.name,
+        )
 
-        # TODO: (DM-28710) Initial implementation of runWEP command in MTAOS
-        raise NotImplementedError("This function is not supported yet (DM-28710).")
+        run_pipetask_cmd += f' -d "exposure IN ({intra_id}, {extra_id})"'
+        run_pipetask_cmd += f" -j {self.pipeline_n_processes}"
+
+        self.log.debug(f"Running: {run_pipetask_cmd}")
+
+        # Run pipeline task in a process asynchronously
+        wep_process = await asyncio.create_subprocess_shell(
+            run_pipetask_cmd,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE,
+        )
+
+        stdout, stderr = await wep_process.communicate()
+
+        self.log.debug(f"Process returned: {wep_process.returncode}")
+
+        self.log.debug(stdout)
+
+        self.log.error(stderr)
+
+        self.log.debug("Data processing completed. Gathering output.")
```

Comment:
Do you want to consider to remove the empty line here to make the `self.log` to be more readable?